blockdev/src/blockdev.rs

```diff
@@ -217,13 +239,54 @@ impl LoopbackDevice {
         self.dev.as_deref().unwrap()
     }
 
+    /// Spawn a cleanup helper process that will clean up the loopback device
+    /// if the parent process dies unexpectedly
+    fn spawn_cleanup_helper(device_path: &str) -> Result<LoopbackCleanupHandle> {
+        use std::os::unix::process::CommandExt;
+        use std::process::Command;
+
+        // Get the path to our own executable
+        let self_exe = std::fs::read_link("/proc/self/exe")
+            .context("Failed to read /proc/self/exe")?;
+
+        // Create the helper process using exec
+        let mut cmd = Command::new(self_exe);
```

Comment:
Set up std{in,out,err} here via https://doc.rust-lang.org/std/process/struct.Stdio.html#method.null